src/screens/crew6/engineeringScreen.cpp

```diff
@@ -240,9 +240,9 @@ void EngineeringScreen::onDraw(sf::RenderTarget& window)
         if (selected_system != SYS_None)
         {
             ShipSystem& system = my_spaceship->systems[selected_system];
-            power_label->setText("Power: " + toNearbyIntString(system.power_level * 100) + "%/" + toNearbyIntString(system.power_request * 100) + "%");
+            power_label->setText(tr("slider", "Power: {percent1}% / {percent2}%").format({{"percent1", toNearbyIntString(system.power_level * 100)}, {"percent2", toNearbyIntString(system.power_request * 100)}}));
```

Comment:
Maybe something a little more descriptive than `percent1` and `percent2`?

Following code `power_level_percent` and `power_request_percent` respectively?

(could even make do without the `power_` prefix, probably could infer from context :) )